google/cloud/internal/openssl_util.cc

```diff
@@ -31,6 +40,118 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace internal {
 
 namespace {
+#ifdef _WIN32
+std::string CaptureWin32Errors() {
```

Comment:
It seems that this is always used as `"some prefix" + CaptureWin32Errors()`, consider changing the API to `FormatWin32Errors(absl::string_view prefix)`

Reply:
Renamed, moved to a separate header, added support for concatenating multiple strings, and also used in the implementation of `SignUsingSha256` as well.